src/commands/txCommands/util/index.js

```diff
@@ -29,9 +29,9 @@ const checkEnvVars = () => {
 
 // local streams
 const readFile$ = filepath =>
-	Rx.Observable
-		.bindNodeCallback(fs.readFile)(filepath)
-		.map(buffer => buffer.toString());
+	Rx.Observable.bindNodeCallback(fs.readFile)(filepath).map(buffer =>
```

Comment:
formatting changes make this diff much more difficult to read. either prettier should be integrated into the repo or formatting changes should be left alone.